controllers/datamoverbackup_controller.go

```diff
@@ -150,5 +153,10 @@ func (r *DataMoverBackupReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 func (r *DataMoverBackupReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&pvcv1alpha1.DataMoverBackup{}).
+		Owns(&snapv1.VolumeSnapshotContent{}).
+		Owns(&snapv1.VolumeSnapshot{}).
+		Owns(&v1.PersistentVolumeClaim{}).
+		Owns(&v1.Pod{}).
```

Comment:
Should we own or watch the secret that we create? Same with ReplicationSource.

Reply:
I thought we decided to not to add secrets for now. Correct me if I got that wrong from the sync up today

Reply:
Yes you're correct. Sorry!